Update configure-send-emails.md (#403)
OutSystems/docs-product: src/extensibility-and-integration/configure-send-emails.md

</div>

> ## OutSystems SMTP Port Options

Reviewer: ```suggestion
## OutSystems SMTP port options
```

> OutSystems supports SMTP ports 25, 465, and 587. Consider using secure ports 465 or 587 to avoid potential issues with port 25 throttling. 

Reviewer: ```suggestion
OutSystems supports SMTP ports 25, 465, and 587. However, to avoid potential throttling issues with port 25,  OutSystems recommends using port 587 - the port for encrypted email transmissions using SMTP Secure (SMTPS). Port 465 is also used sometimes for SMTPS. However, this is an outdated implementation and port 587 should be used if possible. 
```

> Po

Reviewer: ```suggestion
```

> Port 465 offers SMTP over SSL (SMTPS), while port 587 is a recommended alternative for SMTP submission, enhancing reliability and security in email communication within OutSystems.

Reviewer: ```suggestion
Po
```
